### PR TITLE
Switch image save dir from scripts/vagrant/images to build/images (release-0.2)

### PR DESCRIPTION
**Pull request by edwarnicke**
## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .mk/vagrant.mk [hunk 1]
```diff
@@ -50,7 +50,9 @@ vagrant-restart-kubelet:
 
 .PHONY: vagrant-%-load-images
 vagrant-%-load-images:
-	@if [ -e "scripts/vagrant/images/$*.tar" ]; then \
+	@if [ -e "$(IMAGE_DIR)/$*.tar" ]; then \
+		mkdir -p scripts/vagrant/images
```
Comment (denis-tingajkin) [inline .mk/vagrant.mk]: nice

File: .mk/packet.mk [hunk 2]
```diff
@@ -47,16 +47,16 @@ packet-stop:
 
 .PHONY: packet-%-load-images
 packet-%-load-images:
-	@if [ -e "scripts/vagrant/images/$*.tar" ]; then \
+	@if [ -e "$(IMAGE_DIR)/$*.tar" ]; then \
 		pushd scripts/terraform; \
 		echo "Loading image $*.tar to master and worker"; \
-		scp ${SSH_OPTS} ../vagrant/images/$*.tar root@`terraform output master${PACKET_CLUSTER_ID}.public_ip`:~/ & \
-		scp ${SSH_OPTS} ../vagrant/images/$*.tar root@`terraform output worker${PACKET_CLUSTER_ID}_1.public_ip`:~/ 
+		scp ${SSH_OPTS} ../../$(IMAGE_DIR)/$*.tar root@`terraform output master${PACKET_CLUSTER_ID}.public_ip`:~/ & \
```
Comment (denis-tingajkin) [inline .mk/packet.mk]: Should be here `../$(IMAGE_DIR`instead of  `../../$(IMAGE_DIR)`?

Comment (edwarnicke) [reply .mk/packet.mk]: Hmm... we are in ./scripts/terraform, and IMAGE_DIR is relative to ./ ... so we'd need to go back up two directories ../../$(IMAGE_DIR) , right?

But it *should* match in the next line down...

Comment (denis-tingajkin) [reply .mk/packet.mk]: yep, you are right

File: .mk/k8s.mk [hunk 3]
```diff
@@ -26,9 +26,8 @@ CONTAINER_TAG?=master
 # That you could set the CLUSTER_RULES_PREFIX different and introduce
 # a new platform to run on with k8s by adding a new include ${method}.mk
 # and setting CLUSTER_RULES_PREFIX to a different value
-ifeq ($(CLUSTER_RULES_PREFIX),)
-CLUSTER_RULES_PREFIX := vagrant
-endif
```
Comment (nickolaev) [inline .mk/k8s.mk]: There is a double inclusion of kind.mk.

Comment (edwarnicke) [reply .mk/k8s.mk]: Should be fixed now :)

**Pull request merged**